### PR TITLE
Update some scripts

### LINKED ISSUE #4946: Fix some abilities coded with wrong parts living on card
I plan to refactor cards which are done as bad as _Ashnod's Battle Gear_.
(ignoring other variants like _Giant Oyster_ from untapRework branch)

For AI UntapPreference is a bit tricky if only done with GameCommands? :thinking:

Effect might be easier, but then the ContinuousBonus logic might also need more changes if I want to avoid Pump helper in the scripts.

### PR DESCRIPTION
**Pull request by tool4ever**
Closes #4946

### CODE REVIEW

File: forge-gui/res/cardsfolder/s/sirens_ruse.txt [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
-SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ All | Destination$ Battlefield | SubAbility$ DBPirateDraw
-SVar:DBPirateDraw:DB$ Draw | NumCards$ 1 | ConditionCheckSVar$ WasPirate | ConditionSVarCompare$ EQ1 | SubAbility$ DBCleanup
-SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | SubAbility$ DBReinitialize
-SVar:DBReinitialize:DB$ StoreSVar | SVar$ WasPirate | Type$ Number | Expression$ 0
-SVar:WasPirate:Number$0
+A:SP$ ChangeZone | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | Origin$ Battlefield | Destination$ Exile | RememberLKI$ True | Imprint$ True | SubAbility$ DBReturn | SpellDescription$ Exile target creature you control, then return that card to the battlefield under its owner's control. If a Pirate was exiled this way, draw a card. | StackDescription$ SpellDescription
+SVar:DBReturn:DB$ ChangeZone | Defined$ Imprinted | Origin$ All | Destination$ Battlefield | SubAbility$ DBPirateDraw
+SVar:DBPirateDraw:DB$ Draw | NumCards$ 1 | ConditionDefined Remembered | ConditionPresent$ Pirate | SubAbility$ DBCleanup
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/s/sirens_ruse.txt:6]: Easiest fix is to give these lines `StackDescription$ None`

**Comment by Northmoc** [comment]
Nice work, just caught that issue from the initial scripter

Sorry about the typo in `committed` 🤦

File: forge-game/src/main/java/forge/game/player/Player.java [hunk 2]
```diff
@@ -3808,7 +3808,7 @@ public void commitCrime() {
 
     }
 
-    public int getCommitedCrimeThisTurn() {
+    public int getCommittedCrimeThisTurn() {
         return commitedCrimeThisTurn;
     }
     public void setCommitedCrimeThisTurn(int v) {
```
Comment (dracontes) [inline forge-game/src/main/java/forge/game/player/Player.java:3814]: As the one to point it out on Discord, I'd be remiss if I didn't note that lines 3812 and 3814 also have the missing 't' on "committed" typo.

File: forge-gui/res/cardsfolder/f/flamestick_courier.txt [hunk 3]
```diff
@@ -3,10 +3,7 @@ ManaCost:2 R
 Types:Creature Goblin
 PT:2/1
 K:You may choose not to untap CARDNAME during your untap step.
-A:AB$ Pump | Cost$ 2 R T | ValidTgts$ Creature.Goblin | TgtPrompt$ Select target Goblin creature | RememberTargets$ True | AILogic$ ContinuousBonus | SpellDescription$ Target Goblin creature gets +2/+2 and has haste for as long as CARDNAME remains tapped. | StackDescription$ SpellDescription
-S:Mode$ Continuous | Affected$ Creature.IsRemembered | AddPower$ 2 | AddToughness$ 2 | AddKeyword$ Haste
-T:Mode$ Untaps | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ ClearRemembered | Static$ True
-SVar:ClearRemembered:DB$ Cleanup | ClearRemembered$ True
+A:AB$ Pump | Cost$ 2 R T | ValidTgts$ Creature.Goblin | TgtPrompt$ Select target Goblin creature | NumAtt$ 2 | NumDef$ 2 | KW$ Haste | Duration$ UntilUntaps | AILogic$ ContinuousBonus | SpellDescription$ Target Goblin creature gets +2/+2 and has haste for as long as CARDNAME remains tapped. | StackDescription$ SpellDescription
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/f/flamestick_courier.txt:6]: Maybe some of these StackDescriptions might be nice with REP for the target?

Obviously just window dressing

Comment (tool4ever) [reply forge-gui/res/cardsfolder/f/flamestick_courier.txt:6]: not a huge fan
it seems to remove targeting information when theres a matching localization

**Review by Northmoc — APPROVED** [review]
Simply improving the auto StackDescription was an excellent call

I'm so used to current MtG where every card seems to break a template

**Pull request merged**